### ISSUE TITLE
evalution performance is influenced with the batch size

### ISSUE THREAD

**Issue by junchen14** [ArrowLuo/CLIP4Clip#12]
hi, in your evaluation code, before you computes the logits, you also compute the batch-level normalization. the normalization score is influenced by the batch number in fact, which indicates that if give different evaluation batch size, and the performance will be also different. 

so in this case, did you do any ablation study on evaluation batch size?

**Comment by ArrowLuo** [comment]
Hi, @junchen14, Figure 3 (a) in the [paper](https://arxiv.org/pdf/2104.08860.pdf) may be what you need.

**Comment by junchen14** [comment]
no it is not what I meant, what I am trying to say, when I try different batch size during evaluation, it gives me different performance.  so in somehow, the performance is depended on the batch size.  did you observe this?
